dotcom-rendering/src/components/ListenToArticle.importable.tsx

```diff
@@ -7,26 +7,55 @@ import { ListenToArticleButton } from './ListenToArticleButton';
 type Props = {
 	articleId: string;
 };
-export const ListenToArticle = ({ articleId }: Props) => {
-	const [showButton, setShowButton] = useState<boolean>(false);
 
-	const isBridgetCompatible = useIsBridgetCompatible('8.5.1');
+export const formatAudioDuration = (durationInSeconds: number): string => {
+	const hours = Math.floor(durationInSeconds / 3600);
+	const minutes = Math.floor((durationInSeconds % 3600) / 60);
+	const seconds = durationInSeconds % 60;
+
+	if (hours > 3) {
+		return '';
+	}
+
+	const formattedDuration = `${
+		hours > 0 ? `${hours.toString().padStart(2, '0')}:` : ''
+	}${minutes > 0 ? minutes.toString().padStart(2, '0') : '00'}:${
+		seconds > 0 ? seconds.toString().padStart(2, '0') : '00'
+	}`;
+
+	return formattedDuration;
+};
 
+export const ListenToArticle = ({ articleId }: Props) => {
+	const [showButton, setShowButton] = useState<boolean>(true);
```

Comment:
This PR describes adding an audio duration, but this default state has also flipped to `true`? I think this means you now initially show the button in all cases, and then hide it if it's not available? Is that intentional?